tests/prmbackslash_2.prm

```diff
@@ -33,7 +33,7 @@ set Surface pressure\
 set Use years in out\
 put instead of seconds = \
 false  # default: true
-set Nonlinear solver\
+set Nonlinear solver \
```

Comment:
I don't consider this a bug in deal.II. We trim the whitespace at the start of the second line so all required whitespace needs to be in the first line. I guess we made a different choice in ASPECT before (do not trim whitespace). I am okay with changing to the deal.II notation, because it allows us to indent followup lines.